app/views/mountain_view/styleguide/show.html.erb

```diff
@@ -14,7 +14,14 @@
         <h2><%= component_stub.meta_title(@component.title, index) %></h2>
         <div class="mv-component">
           <div class="mv-component__item <%= component_stub.meta_classes %>">
-            <%= render_component @component.name, component_stub.properties.clone %>
+            <% if component_stub.meta_partial.blank? %>
+              <%= render_component @component.name, component_stub.properties.clone %>
+            <% else %>
+              <% content_for_name = "mv_#{@component.name}_#{index}" %>
+              <% content_for :"#{content_for_name}",
```

Comment:
Could theoretically use a layout with yield as well... for some reason I can't use a block though... my server crashes if I use a block with content_for or render layout...

Reply:
Interesting... Yes, that would look much better, IMO. Passing a `mv_content_for_name` feels kinda dirty.

Ideally this would be something like:
```
<%= render component_stub.meta_layout do %>
  <%=  render_component @component.name, component_stub.properties.clone %>
<% end %>
```

And we could generalize it more if we returned an empty partial (with just `yield`) by default in `MountainView::Stub#meta_partial` (and only override if specified), so we don't need the `if component_stub.meta_partial.blank?` check.

Reply:
I like that idea a lot... but I have to figure out why using the render as above causes the server to crash.  Have you heard anything about that?  I lean toward blaming windows :/ But it probably needs to work on windows, so I'd have to figure it out.

Reply:
I haven't heard anything about that :(
But if you want to push those changes that make your server crash either here or to a different branch, I'm more than happy to take a look and try to see if I can reproduce or not.